conans/test/options_in_requirements_test.py

```diff
@@ -18,8 +18,9 @@ def basic_test(self):
 class ConanLib(ConanFile):
     name = "zlib"
     version = "0.1"
-    options = {"shared": [True, False]}
-    default_options= "shared=False"
+    
+    def options(self, config):
+        config.add_shared()
```

Comment:
How does it looks to add other options than shared & standards when using the ``options()`` method?

Reply:
`self.options.add_option("flavour", ["choco", "strawberry", "orange"], "orange")`
It has to be added to the docs, of course